cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -236,52 +235,75 @@ runDbSyncNodeNodeClient iomgr trce plugin nodeConfig (SocketPath socketPath) = d
                           (TraceSendRecv (Handshake NodeToClientVersion CBOR.Term)))
     handshakeTracer = toLogObject $ appendName "Handshake" trce
 
-
-localInitiatorNetworkApplication
-  :: forall blk.
-     (blk ~ ByronBlock)
-  -- TODO: the need of a 'Proxy' is an evidence that blk type is not really
-  -- needed here.  The wallet client should use some concrete type of block
-  -- from 'cardano-chain'.  This should remove the dependency of this module
-  -- from 'ouroboros-consensus'.
-  => Trace IO Text
+    dbSyncVersions
+        :: StrictTMVar IO (GenTx ByronBlock)
+        -> Versions
+             NodeToClientVersion
+             DictVersion
+             (ConnectionId LocalAddress
+                 -> OuroborosApplication 'InitiatorApp BSL.ByteString IO () Void)
+    dbSyncVersions txv = foldMapVersions
+        (\v ->
+          versionedNodeToClientProtocols
+            (nodeToClientProtocolVersion (Proxy @blk) v)
+            (NodeToClientVersionData { networkMagic = nodeNetworkMagic (Proxy @blk) topLevelConfig })
+            (dbSyncProtocols v trce plugin topLevelConfig txv)
+        )
+        (supportedNodeToClientVersions (Proxy @blk))
```

Comment:
@erikd is `cardano-db-sync` using the same config file as `cardano-node`?
If it is, once https://github.com/input-output-hk/cardano-node/pull/830 is merged we can use the specified versions rather than `supportedNodeToClientVersions`.

Reply:
no, the configs are different.